wagn/features/notifications.feature

```diff
@@ -19,7 +19,8 @@ Feature: Notifications
     Then Joe Admin should be notified that "Joe User created \"Foo\""
     And the card Phrase+*type+*followers should point to "Joe Admin"
     And I should see "was just created by Joe User" in the email body
-    And I should see |You received this email because you're following "all "Phrases""| in the email body  #FIXME these double quotes are ugly
+    #FIXME these double quotes are ugly
+    And I should see |You received this email because you're following "all "Phrases""| in the email body
```

Comment:
It wasn't matching the step definition.  I don't think cucumber strips comments before matching, so the comment has to be on it's own line.  That appears to be the quoting generated, so you have to fix the test, then fix the code.  For now, these steps actually pass, and I don't really know what you want the fixed behavior to be.